jobs/test-override.Jenkinsfile

```diff
@@ -201,11 +223,47 @@ try {
             def arch = architecture
             parallelruns[arch] = {
                 pipeutils.withOptionalExistingCosaRemoteSession(arch: arch, session: archinfo[arch]['session']) {
+                    // Download Koji Build rpms
                     for (id in koji_build_ids) {
                         // XXX: I think this will fail for a package that doesn't have noarch
                         // packages nor packages on $arch
                         shwrap("cosa shell -- env -C overrides/rpm koji download-build ${id} --arch ${arch} --arch noarch")
                     }
+                    // Download Koji Task (scratch) rpms
+                    for (id in koji_scratch_build_ids) {
+                        shwrap("cosa shell -- env -C overrides/rpm koji download-task ${id} --arch ${arch} --arch noarch --skip='debug|test'")
+                    }
+                    // Download copr build rpms
+                    for (id in copr_build_ids) {
```

Comment:
There is a lot of work done in this code block that maybe we don't have to do manually ourselves.. What if we installed `copr-cli` in COSA and just use [copr-cli download-build](https://www.mankier.com/1/copr-cli#Build_Actions-copr-cli_download-build_%5Boptions%5D):

```
cosa shell -- copr-cli download-build --chroot ${chrootname} --dest overrides/rpm --chroot ${id}
```

This will make things better for us over time if they say change their API we shouldn't have to update anything.

Reply:
Yes, that would be easier, thought might add 12 more packages (6 Mib) and make coreos-assembler bigger. 
```
[coreos-assembler]$ sudo dnf install copr-cli
Transaction Summary:
 Installing:        12 packages

Total size of inbound packages is 1 MiB. Need to download 1 MiB.
After this operation, 6 MiB extra will be used (install 6 MiB, remove 0 B).
```

Reply:
to me COSA is massive anyway so 6 MiB isn't that big of a deal.

Reply:
Add copr-cli in PR https://github.com/coreos/coreos-assembler/pull/4086